src/index.html

```diff
@@ -0,0 +1,214 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="./style.css">
+    <title>Human rights</title>
+</head>
+<body>
+
+    <h1>
```

Comment:
I think it will be better
```suggestion
    <h1>Universal Declaration of Human Rights</h1>
```